gitpod-io/website
# feat: Newsletter signup under each blog & guide

## Description

This PR adds the Newsletter signup component under each blog & guides with custom text.

|Component|Blog|Guides|
|:---:|:---:|:---:|
|<img width="649" alt="image" src="https://user-images.githubusercontent.com/55068936/214480337-3baf2b37-79b1-4c22-8fd9-77d6a1b38e38.png">|<img width="816" alt="image" src="https://user-images.githubusercontent.com/55068936/214480305-b4ee3204-6b48-4f84-87db-75957b652c15.png">|<img width="750" alt="image" src="https://user-images.githubusercontent.com/55068936/214480367-4b915dc0-31dd-41d8-8bcf-d2391db20008.png">|

## Related Issue(s)



Fixes #3273

## How to test

See the Following Previews:

- [`/guides/custom-java-distribution-on-gitpod`](https://deploy-preview-3296--gitpod-kumquat.netlify.app/guides/custom-java-distribution-on-gitpod)
- [`/blog/better-container-development-on-apple-m1-macbooks-with-gitpod`](https://deploy-preview-3296--gitpod-kumquat.netlify.app/blog/better-container-development-on-apple-m1-macbooks-with-gitpod)
